authors/apps/profiles/tests/test_base.py

```diff
@@ -0,0 +1,19 @@
+from django.urls import reverse
+from rest_framework.test import APITestCase
+from .test_data import new_user
+from authors.apps.authentication.models import User
```

Comment:
hey @wasibani-roy,  it looks like you are not using this import ,you can always remove all unused comments to keep the code clean